mex/backend/ingest/main.py

```diff
@@ -10,6 +10,6 @@
 def ingest_extracted_items(request: BulkIngestRequest) -> BulkIngestResponse:
     """Ingest batches of extracted items grouped by their type."""
     connector = GraphConnector.get()
-    models = request.get_all()
+    models = request.items
     identifiers = connector.ingest(models)
```

Comment:
could even simplify this to

```suggestion
    identifiers = connector.ingest(request.items)
```